dashboard/test/ui/features/learning_platform/projects/applab_project.feature

```diff
@@ -103,8 +103,6 @@ Scenario: Save Project After Signing Out
   When I sign out using jquery
   And I add code "// comment 2" to ace editor
   And ace editor code is equal to "// comment 1// comment 2"
```

Comment:
Should probably remove line 104 and 105 too.

Reply:
Is that important to show that the code entered after sign out didn't get saved to their account?

Reply:
Without the reset button click, we can't definitively trigger a save. This seems like an odd test since the page immediately redirects on sign out - a student would never get an opportunity to continue editing their code.

Reply:
It might make sense to change this to test that the page redirects. I'm guessing this was testing old behavior.

Reply:
I take that back. I didn't realize what "sign out with jquery" does. (no automatic page reload)